### ISSUE TITLE
global leak with requestAnimationFrame, cancelAnimationFrame

### ISSUE THREAD

**Issue by NoxWings** [sinonjs/lolex#144]
I've already opened an issue in Sinon about a leak being reported by mocha while using sinon fake timers.
https://github.com/sinonjs/sinon/issues/1617

To replicate the error add --check-leaks to the mocha testing command on the `package.json`
```diff
  "scripts": {
    "lint": "eslint .",
-    "test-node": "mocha -R dot",
+    "test-node": "mocha -R dot --check-leaks",
    "test-headless": "mochify",
    "test-cloud": "mochify --wd",
    "test": "npm run lint && npm run test-node && npm run test-headless",
    "bundle": "browserify -s lolex -o lolex.js src/lolex-src.js",
    "prepublish": "npm run bundle",
    "precommit": "run-p lint test-node"
  }
```

output
```
1) lolex tick "before each" hook:
     Error: global leaks detected: requestAnimationFrame, cancelAnimationFrame
```

I think this is probably the PR that introduced the bug, as this has only been failing for the last day or so and this is the only PR which mentions requestAnimationFrame.
https://github.com/sinonjs/lolex/pull/137/files

**Comment by fatso83** [comment]
Great, thanks. Already reported as #142 a few minutes earlier, and a fix is in!

**Comment by fatso83** [comment]
Released as 2.3.0
